src/api/spec/models/bs_request_spec.rb

```diff
@@ -138,6 +139,57 @@
     end
   end
 
+  describe '#change_review_state' do
+    let(:user) { create(:confirmed_user) }
+    let!(:request) { create(:bs_request, creator: user.login) }
+    let(:reviewer) { create(:confirmed_user) }
+    let(:someone) { create(:confirmed_user) }
+
+    context 'with by_user review' do
+      before do
+        User.current = user
+        request.addreview(by_user: reviewer, comment: 'does it look ok?')
+      end
+
+      it 'raises exception on missing by_ paramter' do
+        expect do
+          request.change_review_state(:accepted)
+        end.to raise_error(BsRequest::InvalidReview)
```

Comment:
That's more about personal taste, but I would prefer to write this in one line unless it exceeds the limit, like 
` expect { request.change_review_state(:accepted) }.to raise_error(BsRequest::InvalidReview) `